services/treeware/treeware-trees.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const crypto = require('crypto')
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { floorCount } = require('../color-formatters')
+const { BaseJsonService } = require('..')
+
+const apiSchema = Joi.object({
+  total: Joi.number().required(),
+}).required()
+
+module.exports = class TreewareTrees extends BaseJsonService {
+  static get category() {
+    return 'other'
```

Comment:
We also have a `funding`category, Not sure which one is more appropriate, what do you think?

Reply:
I wasn't sure about this, I guess funding is probably more appropriate. 👍

@jamesmills, I was just interested what you'd suggest for this?

Reply:
I think that `funding` would be okay, but I want to throw in `license` as well. Because it's part of the license and not an optional funding.
The point is that every user is **required** to be trees if he uses the Treeware licensed software in production.

Reply:
We actually leave it to the package maintainer to decide how to work the Treeware "condition of use". We currently have two in the wild.

> This package is free and open-source, under the MIT license. Use it however you want. This package is Treeware. If you use it in production, then we ask that you buy the world a tree to thank us for our work.

> You're free to use this package, but if it makes it to your production environment you are required to buy the world a tree.

In the first example, it's attached to the licence. However, the maintainer can decide how to word it.

I think it's a licence and would be happiest with this option.